report/renderer/i18n.js

```diff
@@ -177,19 +201,33 @@ export class I18n {
     /** @type {Array<string>} */
     const parts = [];
     /** @type {Record<string, number>} */
-    const unitLabels = {
-      d: 60 * 60 * 24,
-      h: 60 * 60,
-      m: 60,
-      s: 1,
+    const unitToSecondsPer = {
+      day: 60 * 60 * 24,
+      hour: 60 * 60,
+      minute: 60,
+      second: 1,
+    };
+    /** @type {Record<string, string>} */
+    const unitToDefaultLabel = {
+      day: 'd',
+      hour: 'h',
+      minute: 'm',
+      second: 's',
     };
 
-    Object.keys(unitLabels).forEach(label => {
-      const unit = unitLabels[label];
-      const numberOfUnits = Math.floor(timeInSeconds / unit);
+    Object.keys(unitToSecondsPer).forEach(unit => {
+      const unitFormatter = new Intl.NumberFormat(this._locale, {
+        style: 'unit',
+        unit,
+        unitDisplay: 'narrow',
+      });
+      const label = unitFormatter.formatToParts(0)
```

Comment:
this seems like it could be problematic to use 0 here for languages with different (abbreviated) unit names for plurals? Might not be a problem for any of our languages, but this also seems to be a very roundabout way vs concatenating calls to `_formatNumberWithGranularity`. What's the motivation? May be worth a comment

Reply:
<img width="678" alt="image" src="https://user-images.githubusercontent.com/4071474/162335973-301b6513-a824-42dc-bdbe-3595fd9daff8.png">

not all locales have a narrow display for every unit... not even english.

Reply:
good point, better to call that function directly